Exiled.API/Enums/EffectType.cs

```diff
@@ -199,5 +199,25 @@ public enum EffectType
         /// Teleports the player to the pocket dimension and drains health until the player escapes or is killed.
         /// </summary>
         PocketCorroding,
+
+        /// <summary>
+        /// Reduces walking sound by 10%.
+        /// </summary>
+        SilentWalk,
+
+        /// <summary>
+        /// Makes you a marshmallow guy.
+        /// </summary>
+        Marshmallow,
```

Comment:
```suggestion
        [Obsolete("Not functional in-game")]
        Marshmallow,
```
basically what Yamato is saying is